.github/workflows/reproduce-schedule.yml

```
@@ -23,6 +23,10 @@ jobs:
           grep -i "tag_name" | \
           awk -F \" '{print "TAG="$4}' >> $GITHUB_ENV
 
+      - name: Free up disk space
+        run: |
+          sudo rm -rf /usr/share/dotnet /usr/local/lib/android /opt/ghc /opt/hostedtoolcache /opt/microsoft /opt/pipx /usr/local/julia* /usr/lib/jvm /usr/lib/google-cloud-sdk /usr/lib/llvm* /usr/lib/firefox /usr/share/swift /usr/share/miniconda /usr/share/az* /usr/share/gradle* /usr/lib/python* /usr/local/share/powershell /usr/local/share/chromium /usr/local/share/vcpkg
+
```

Let's reuse our cleanup script from [`build.yml`](https://github.com/simplex-chat/simplex-chat/blob/stable/.github/workflows/build.yml#L150-L154).

```suggestion
      # Otherwise we run out of disk space with Docker build
      - name: Free disk space
        shell: bash
        run: ./scripts/ci/linux_util_free_space.sh
```